website/src/pages/download.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+
+import { Column, Container, Row } from '../components/Grid';
+import Sponsors from '../components/Sponsors';
+
+
+const LATEST_VERSION = '5.0.0-alpha15';
+
+
+const DownloadLink = ({ arch }) => {
+	let filename, label;
+	switch (arch) {
+	case 'macos':
+		filename = 'macosx.tar.gz';
+		label = 'MacOS';
+		break;
+	case 'linux':
+		filename = 'linux.tar.gz';
+		label = 'Linux';
+		break;
+	case 'src':
+		filename = 'src.zip';
+		label = 'Source Code';
+		break;
+	case 'windows':
+		filename = 'windows.zip';
+		label = 'Windows';
+		break;
+	}
+	return (
+		<div className="download-link">
+			<Link to={`https://github.com/premake/premake-core/releases/download/v${LATEST_VERSION}/premake-${LATEST_VERSION}-${filename}`}>
+				<b>{label}</b>
+			</Link>
+		</div>
+	);
+};
+
+
+const Download = () =>
+	<Layout title="Download">
+		<main className="download">
+			<Container className="intro">
+				<Row>
+					<Column>
+						<h1>Download Premake</h1>
+						<p>
+							Premake is a self-contained, single file command line executable which should build and run pretty much everywhere.
+							See <Link to="/docs/using-premake">Using Premake</Link> for usage instructions and help getting started.
+						</p>
+						<p>
+							The latest released version is <b>v{LATEST_VERSION}</b>. <Link to="https://github.com/premake/premake-core/releases">See all releases</Link>.
+						</p>
+					</Column>
+				</Row>
+			</Container>
+			<Container>
+				<Row>
+					<Column>
+						<h3>Pre-Built Binaries</h3>
+						<p>Binaries simply need to be unpacked and placed somewhere on the system search path or any other convenient location.</p>
+						<p>
+							<DownloadLink arch="windows" /><br />
+							<DownloadLink arch="linux" /><br />
+							<DownloadLink arch="macos" /><br />
+						</p>
```

Comment:
`<div>` cannot appear as a descendant of `<p>`. You will see this error when you go to download page and open console in your browser. It's not react error. Website is compiled correctly but accorting to W3C you cannot put *block elements* inside `<p>` and React is kind enough to report this in the console.

My suggestion here is to change this paragraph with `DownloadLink`s into list `<ul>`, `<li>` then add a class to `<ul>`, e.g. "download-links" and style it. (then you can remove "download-link" class from `DownloadLink`). Unlike `<p>`, lists can contain *divs* and other *block elements*.

* To remove list bullets use: `list-style: none;`
* You can also style each item in the list: `.download-links > li { /*...*/ }`